administrator/language/de-DE/plg_fields_number.ini

```
@@ -0,0 +1,20 @@
+; Joomla! German Translation
+; (C) 2025 Open Source Matters, Inc. <https://www.joomla.org>
+; (C) Translation 2008 - 2025 J!German <https://www.jgerman.de>
+; License GNU General Public License version 2 or later; see LICENSE.txt
+; Note : All ini files need to be saved as UTF-8
+
+PLG_FIELDS_NUMBER="Felder - Zahl"
+PLG_FIELDS_NUMBER_LABEL="Zahl (%s)"
+PLG_FIELDS_NUMBER_PARAMS_CURRENCY_LABEL="Als Währung formatieren"
+PLG_FIELDS_NUMBER_PARAMS_DECIMALS_LABEL="Anzahl der Dezimalstellen"
+PLG_FIELDS_NUMBER_PARAMS_MAX_DESC="Maximaler Wert, der ausgewählt werden kann (beschränkt nur die Pfeiltasten)."
```

```suggestion
PLG_FIELDS_NUMBER_PARAMS_MAX_DESC="Der größte Wert, der ausgewählt werden kann (beschränkt nur die Pfeiltasten)."
```
Im Vergleich zu PLG_FIELDS_NUMBER_PARAMS_MIN_DESC

Alternativ von MDN: `Der maximale Wert, der für diese Eingabe akzeptiert wird.`